tests/lib/openstack_query/handlers/test_client_side_handler.py

```diff
@@ -0,0 +1,207 @@
+import unittest
+from unittest.mock import MagicMock, patch
+from parameterized import parameterized
+
+from openstack_query.handlers.client_side_handler import ClientSideHandler
+from exceptions.query_preset_mapping_error import QueryPresetMappingError
+
+from tests.lib.openstack_query.mocks.mocked_query_presets import MockQueryPresets
+from tests.lib.openstack_query.mocks.mocked_props import MockProperties
+
+
+class ClientSideHandlerBaseTests(unittest.TestCase):
+    """
+    Runs various tests to ensure that ClientSideHandlerBase class methods function expectedly
+    """
+
+    def setUp(self):
+        """
+        Setup for tests
+        """
+        super().setUp()
+
+        _FILTER_FUNCTION_MAPPINGS = {
+            MockQueryPresets.ITEM_1: ["*"],
+            MockQueryPresets.ITEM_2: [MockProperties.PROP_1, MockProperties.PROP_2],
+            MockQueryPresets.ITEM_3: [MockProperties.PROP_3, MockProperties.PROP_4],
+        }
+        self.instance = ClientSideHandler(_FILTER_FUNCTION_MAPPINGS)
+        self.instance._FILTER_FUNCTIONS = {
+            MockQueryPresets.ITEM_1: "item1_func",
+            MockQueryPresets.ITEM_2: "item2_func",
+            MockQueryPresets.ITEM_3: "item3_func",
+        }
```

Comment:
If we're having to mess with the object's privates, maybe we should have a `FakeClientSideHandler` or similar?

Reply:
what do you mean by that? 
Here we are testing `ClientSideHandler` - we mock a set of filter function mappings that would be passed as input to the constructor
I don't see how a `FakeClientSideHandler` would help here?